src/palette/internal/palettecreator.cpp

```diff
@@ -1530,65 +1530,35 @@ PalettePtr PaletteCreator::newTextPalette(bool defaultPalette)
-
-    auto no = makeElement<PlayTechAnnotation>(gpaletteScore);
-    //: For brass and plucked string instruments: staff text that prescribes to play without mute, see https://en.wikipedia.org/wiki/Mute_(music)
-    no->setXmlText(QT_TRANSLATE_NOOP("palette", "open"));
-    no->setTechniqueType(PlayingTechniqueType::Open);
-    //: For brass and plucked string instruments: staff text that prescribes to play without mute, see https://en.wikipedia.org/wiki/Mute_(music)
-    sp->appendElement(no, QT_TRANSLATE_NOOP("palette", "Open"))->setElementTranslated(true);
-
-    auto distort = makeElement<PlayTechAnnotation>(gpaletteScore);
-    distort->setXmlText(QT_TRANSLATE_NOOP("palette", "distort"));
-    distort->setTechniqueType(PlayingTechniqueType::Distortion);
-    sp->appendElement(distort, QT_TRANSLATE_NOOP("palette", "Distortion"))->setElementTranslated(true);
-
-    auto overdrive = makeElement<PlayTechAnnotation>(gpaletteScore);
-    overdrive->setXmlText(QT_TRANSLATE_NOOP("palette", "overdrive"));
-    overdrive->setTechniqueType(PlayingTechniqueType::Overdrive);
-    sp->appendElement(overdrive, QT_TRANSLATE_NOOP("palette", "Overdrive"))->setElementTranslated(true);
-
-    auto harmonics = makeElement<PlayTechAnnotation>(gpaletteScore);
-    harmonics->setXmlText(QT_TRANSLATE_NOOP("palette", "harmonics"));
-    harmonics->setTechniqueType(PlayingTechniqueType::Harmonics);
-    sp->appendElement(harmonics, QT_TRANSLATE_NOOP("palette", "Harmonics"))->setElementTranslated(true);
-
-    auto jazzTone = makeElement<PlayTechAnnotation>(gpaletteScore);
-    jazzTone->setXmlText(QT_TRANSLATE_NOOP("palette", "jazz tone"));
-    jazzTone->setTechniqueType(PlayingTechniqueType::JazzTone);
-    sp->appendElement(jazzTone, QT_TRANSLATE_NOOP("palette", "Jazz tone"))->setElementTranslated(true);
-
-    auto normal = makeElement<PlayTechAnnotation>(gpaletteScore);
-    normal->setXmlText(QT_TRANSLATE_NOOP("palette", "normal"));
-    normal->setTechniqueType(PlayingTechniqueType::Natural);
-    sp->appendElement(normal, QT_TRANSLATE_NOOP("palette", "Normal"))->setElementTranslated(true);
+    struct PlayTechAnnotationInfo {
+        muse::TranslatableString xmlText;
+        PlayingTechniqueType playTechType;
+        bool expressionType = false;
+    };
+
+    static const std::vector<PlayTechAnnotationInfo> playTechAnnotations = {
+        { muse::TranslatableString("palette", "legato"),    PlayingTechniqueType::Legato },
+        { muse::TranslatableString("palette", "pizz."),     PlayingTechniqueType::Pizzicato },
+        { muse::TranslatableString("palette", "arco"),      PlayingTechniqueType::Natural },
+        { muse::TranslatableString("palette", "tremolo"),   PlayingTechniqueType::Tremolo, true },
+        { muse::TranslatableString("palette", "mute"),      PlayingTechniqueType::Mute }, //: For brass and plucked string instruments: staff text that prescribes to use mute while playing, see https://en.wikipedia.org/wiki/Mute_(music)
```

Comment:
The `//:` comments will need to be on the line above the line that contains the string. Otherwise `lupdate` doesn't detect it or relates it to the wrong string.

(these comments namely have a special meaning: when extracting translatable strings from the source code, these comments are extracted too, and will appear as hints for the translators on Transifex.)